src/AutoRest.CSharp/Common/Output/Models/Shared/CSharpAttribute.cs

```diff
@@ -1,9 +1,10 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License. See License.txt in the project root for license information.
 
+using AutoRest.CSharp.Common.Output.Expressions.ValueExpressions;
 using AutoRest.CSharp.Generation.Types;
 
 namespace AutoRest.CSharp.Output.Models.Shared
 {
-    internal record CSharpAttribute(CSharpType Type, params string[] Arguments);
+    internal record CSharpAttribute(CSharpType Type, params ValueExpression[] Arguments);
```

Comment:
well if possible let us take this oppotunity to change this to `IReadOnlyList<ValueExpression>`?